### ISSUE TITLE
add datamgr code to create and populate tables for system tests

### ISSUE THREAD

**Issue by tswast** [ibis-project/ibis-bigquery#51]
See: https://github.com/ibis-project/ibis/blob/98e2cd84478f9c8325879bb5b77527a23145f419/ci/datamgr.py#L484-L617 from ibis-framework 1.4.0

**Comment by tswast** [comment]
We might want to do this as part of the test fixtures, instead. Either way, we need some way of populating these data.

**Comment by loudinb** [comment]
@tswast I started to look at this while getting my dev environment set up.  Ok to proceed implementing this to get the test data populated?

```python
@pytest.fixture(scope='session')
def bqclient(client):
    return client.client


@pytest.fixture(scope='session')
def testing_dataset(bqclient):
    dataset_ref = bigquery.DatasetReference(bqclient.project, DATASET_ID)
    try:
        bqclient.get_dataset(dataset_ref)
    except NotFound:
        bqclient.create_dataset(dataset_ref)
    yield dataset_ref
    bqclient.delete_dataset(dataset_ref, delete_contents=True)


@pytest.fixture(scope='session')
def date_table(testing_dataset):
    return bigquery.TableReference(testing_dataset, 'date_column_parted')


@pytest.fixture(autouse=True, scope='session')
def create_date_table(bqclient, date_table):
    table = bigquery.Table(date_table)
    table.schema = [
        bigquery.SchemaField('my_date_parted_col', 'DATE'),
        bigquery.SchemaField('string_col', 'STRING'),
        bigquery.SchemaField('int_col', 'INTEGER'),
    ]
    table.time_partitioning = bigquery.TimePartitioning(
        field='my_date_parted_col'
    )
    bqclient.create_table(table, exists_ok=True)


@pytest.fixture(scope='session')
def timestamp_table(testing_dataset):
    return bigquery.TableReference(testing_dataset, 'timestamp_column_parted')


@pytest.fixture(autouse=True, scope='session')
def create_timestamp_table(bqclient, timestamp_table):
    table = bigquery.Table(timestamp_table)
    table.schema = [
        bigquery.SchemaField('my_timestamp_parted_col', 'DATE'),
        bigquery.SchemaField('string_col', 'STRING'),
        bigquery.SchemaField('int_col', 'INTEGER'),
    ]
    table.time_partitioning = bigquery.TimePartitioning(
        field='my_timestamp_parted_col'
    )
    bqclient.create_table(table, exists_ok=True)
```

**Comment by tswast** [comment]
@loudinb Fixtures like that in tests/system/conftest.py sound perfect.

**Comment by tswast** [comment]
Closed by #60
